Gemfile

```diff
@@ -18,37 +18,18 @@ gem 'jc-validates_timeliness'
 gem 'jquery-rails'
 gem 'kaminari'
 gem 'nokogiri'
-gem 'pg', '< 1.0'
+gem 'pg'
 gem 'phonelib'
-gem 'rails', '5.1.7'
-gem 'responders', '~> 2.0'
+gem 'puma' # Heroku+Puma docs: https://bit.ly/3blHFc7
+gem 'rails', '~> 5.2.6'
 gem 'sassc-rails' # see https://github.com/sass/sassc-rails/issues/114
-gem 'sprockets', '~> 3.7.2' # see https://github.com/rails/sprockets/issues/632
-gem 'thin'
-gem 'uglifier', '>= 1.3.0'
+gem 'sprockets', '< 4' # v4 has multiple issues
+gem 'uglifier'
 gem 'redcarpet'
-gem 'unf' # <- fog <- asset_sync (http://bit.ly/17TiMjA)
 gem 'twilio-ruby'
 gem 'prettier'
 gem 'mini_magick'
 
-# Groups: Rails will load the group where name == Rails.env
-# http://yehudakatz.com/2010/05/09/the-how-and-why-of-bundler-groups/
-#
-# To keep your heroku slug size down, heroku automatically
-# bundles --without development:test
-# https://blog.heroku.com/archives/2011/2/15/using-bundler-groups-on-heroku
-
-# Heroku gems hopefully replace plugins.
-# https://devcenter.heroku.com/articles/rails4#heroku-gems
-# Unfortunately, heroku still injects plugins.
-# But that should be fixed soon by:
-# https://github.com/heroku/heroku-buildpack-ruby/pull/11
-# Then again, do I really want serve_static_assets?
-group :production, :stage do
-  gem 'rails_12factor'
-end
-
 # rspec-rails wants to be in the :development group
 # to "expose generators and rake tasks"
 group :test, :development do
```

Comment:
When I checked this out, updated my ruby and bundle versions, and tried to run `rails server`, I got a crash with an error about needing to add the gem "listen". I added it to the `:development` group here, and it fixed the issue. Were you able to run things locally without that?

Reply:
Good catch. I have been using the `test` environment, so this didn't come up. Added in 3d31cd4.